src/state/contexts/UsersContext.js

```diff
@@ -4,7 +4,7 @@ export const UsersContext = createContext({});
 
 const UsersProvider = ({ children }) => {
   const [userRole, setUserRole] = useState();
-  const [userInfo, setUserInfo] = useState(null);
+  const [userInfo, setUserInfo] = useState();
```

Comment:
how do we receive the userInfo back from postman or insomnia? does this state need to be set as an empty array or object? or something else?

Reply:
Thanks. I forgot about this. It was one of the first changes I made when testing for the source of the customer dashboard crash and I didn't remember to revert it. I just corrected that.

Regarding Insomnia, my calls to the database have been working. The format of the calls had to be tweaked to match what the endpoint was expecting for some and they returned the data that was requested. This userInfo in the UsersContext was set up before our team worked on this project and appears to be functioning properly, with the exception of losing userInfo.sub from state during a refresh and causing a customer dashboard crash.

If there's time, I intend to find that bug and squash it.